chalk-solve/src/lib.rs

```diff
@@ -84,6 +84,9 @@ pub trait RustIrDatabase<I: Interner>: Debug {
     /// Returns id of a trait lang item, if found
     fn well_known_trait_id(&self, well_known_trait: WellKnownTrait) -> Option<TraitId<I>>;
 
+    /// Returns id of a trait lang item, if found
+    fn is_trait_object_safe(&self, trait_id: TraitId<I>) -> bool;
+
```

Comment:
Um, object safety should already be implemented as of #445, try pulling the latest code:
https://github.com/rust-lang/chalk/blob/37a38f95bb55c0d91d5e83da3d8628761ccc939a/chalk-solve/src/lib.rs#L98